sdk/src/Services/S3/Custom/Model/Internal/MarshallTransformations/AbortMultipartUploadRequestMarshaller.cs

```diff
@@ -50,6 +50,20 @@ public IRequest Marshall(AbortMultipartUploadRequest abortMultipartUploadRequest
             
             return request;
         }
+
+	    private static AbortMultipartUploadRequestMarshaller _instance;
```

Comment:
If I recall correctly I used this pattern because it was already present in the S3 implementation. So I sticked to what is in the legacy and not to what is in the generated code. If you dislike it feel free to merge my changes into a PR which changes the formatting to your preferred style.